src/Extension.ml

```diff
@@ -11,6 +11,7 @@ module Client = struct
         [| { scheme = "file"; language = "ocaml" }
          ; { scheme = "file"; language = "reason" }
         |]
+    ; revealOutputChannelOn = Some Never
```

Comment:
`jsConverter` isn't automatic, unfortunately
https://bucklescript.github.io/docs/en/generate-converters-accessors#convert-between-js-integer-enum-and-bs-variant